lib/mongo/server.rb

```diff
@@ -161,9 +156,34 @@ def compressor
                    :replica_set_name,
                    :secondary?,
                    :standalone?,
-                   :unknown?,
                    :last_write_date,
-                   :logical_session_timeout
+                   :unknown?
+
+    # @!method features
+    #   @deprecated
+    #
+    # @!method logical_session_timeout
+    #   @deprecated
+    #
+    # @!method max_bson_object_size
+    #   @deprecated
+    #
+    # @!method max_message_size
+    #   @deprecated
+    #
+    # @!method max_wire_version
+    #   @deprecated
+    #
+    # @!method max_write_batch_size
+    #   @deprecated
+    def_delegators :description,
+     :features,
+     :logical_session_timeout,
+     :max_bson_object_size,
+     :max_message_size,
+     :max_wire_version,
+     :max_write_batch_size
```

Comment:
These are all the delegated methods that are no longer used by this class. The ones I didn't move over here are still being used in server selection. See my question in the PR description -- should I change the server selection layer at all?